symfony/webpack-encore-bundle/1.6/config/packages/webpack_encore.yaml

```diff
@@ -0,0 +1,26 @@
+webpack_encore:
+    # The path where Encore is building the assets - i.e. Encore.setOutputPath()
+    output_path: '%kernel.project_dir%/public/build'
+    # If multiple builds are defined (as shown below), you can disable the default build:
+    # output_path: false
+
+    # if using Encore.enableIntegrityHashes() and need the crossorigin attribute (default: false, or use 'anonymous' or 'use-credentials')
+    # crossorigin: 'anonymous'
+
+    # preload all rendered script and link tags automatically via the http2 Link header
+    # preload: true
+
+    # Throw an exception if the entrypoints.json file is missing or an entry is missing from the data
+    # strict_mode: false
+
+    # if you have multiple builds:
+    # builds:
+    # pass "frontend" as the 3rg arg to the Twig functions
+    # {{ encore_entry_script_tags('entry1', null, 'frontend') }}
+
+    # frontend: '%kernel.project_dir%/public/frontend/build'
```

Comment:
This should be indented under `builds`, right? The formatting in this section with the comments confuses me (might just be some missing indentation)